e2e/process-services-cloud/people-group-cloud-filter-component.e2e.ts

```diff
@@ -66,8 +66,10 @@ xdescribe('People Groups Cloud Component', () => {
             clientActivitiAdminRoleId = await rolesService.getClientRoleIdByRoleName(groupActiviti.id, clientId, CONSTANTS.ROLES.ACTIVITI_ADMIN);
             clientActivitiUserRoleId = await rolesService.getClientRoleIdByRoleName(groupActiviti.id, clientId, CONSTANTS.ROLES.ACTIVITI_USER);
 
-            apsUser = await identityService.createIdentityUser();
+            testUser = await identityService.createIdentityUser();
             apsUserRoleId = await rolesService.getRoleIdByRoleName(CONSTANTS.ROLES.APS_USER);
```

Comment:
If those precondition are not needed for all the tests, can we move each precondition in the specific test?
Instead of being in the `afterall` move in the `it`

Reply:
These preconditions are needed before the all the tests are run. They are not specific to a particular test here.